libgloss/arc/hl/hl_fstat.c

```diff
@@ -0,0 +1,80 @@
+/*********************************************************************
+Copyright  2002-2020 Synopsys Inc.
+*********************************************************************/
+
+#include <errno.h>
+#include <stdarg.h>
+#include <stdint.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+#include "hl_toolchain.h"
+#include "hl_api.h"
+
+#pragma pack (push, 4)
```

Comment:
GCC doesn't support very good `#pragma`  please remove it and use attributes

Reply:
done